examples/animation/bayes_update.py

```diff
@@ -41,6 +41,10 @@ def __init__(self, ax, prob=0.5):
     def __call__(self, i):
         # This way the plot can continuously run and we just keep
         # watching new realizations of the process
+        if i == 0:
```

Comment:
perhaps mention that this is needed so that no error occurs *when the animation loops*?  Otherwise this would seem unnecessary

Reply:
That's what the comment that was there is about?

Reply:
ah, yes